models/optional_modules/consent/snowplow_web_consent_cmp_stats.sql

```diff
@@ -5,6 +5,53 @@
   )
 }}
 
+{%- if target.type in ('postgres') -%}
```

Comment:
Does this also need to say redshift?

Reply:
No, I tested it locally with Redshift and it passed. Docs also say it accepts the ignore nulls bit: https://docs.aws.amazon.com/redshift/latest/dg/r_WF_first_value.html